packages/model-viewer/src/test/features/annotation-spec.ts

```diff
@@ -132,11 +132,11 @@ suite('ModelViewerElementBase with AnnotationMixin', () => {
         let wrapper: HTMLElement;
 
         setup(async () => {
+          await rafPasses();
```

Comment:
I'm sure I put this here in the first place, but can we leave a note as to its significance?

Reply:
Done.